src/lib/components/Search.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { createEventDispatcher } from "svelte";
+
+    let searchTerm = '';
+
+    const dispatch = createEventDispatcher();
+
+    function onSubmit(){
+        dispatch("search", {searchTerm})
+    }
+
+</script>
+
+<form on:submit|preventDefault={onSubmit} class="p-4 space-y-4">
+	<label for="search" />
+	<input
+		type="text"
+		bind:value={searchTerm}
+		placeholder="Search by keyword"
+		id="search"
+		name="search"
+		class="p-2 rounded-lg border-2 border-green-500 dark:border-green-700 text-green-700 dark:text-green-500"
```

Comment:
it is missing a background colour for dark mode and doesn't align left with the rest of the content:
<img width="542" alt="Screen Shot 2023-08-02 at 8 58 39 PM" src="https://github.com/ClimateTown/knowledge-hub/assets/4835191/7e47d4c3-191f-4dad-b105-16eeb4d5b99e">

Reply:
Add background color from white to transparent. Align form to the left by removing padding and adding margin to the bottom.